ddtrace/tracer/tracer.go

```diff
@@ -171,9 +224,11 @@ func (t *tracer) worker() {
 			t.pushPayload(trace)
 
 		case <-ticker.C:
+			t.config.statsd.Incr("datadog.tracer.flush.count", []string{"reason:scheduled"}, 1)
```

Comment:
Can we apply the namespace policy everywhere? Choose `datadog.trace.flush_count`.